demisto_sdk/commands/split/tests/jsonsplitter_test.py

```diff
@@ -48,3 +55,49 @@ def test_split_json(repo):
             result_generic_module = json.load(f)
 
         assert result_generic_module == GENERIC_MODULE
+
+
+@pytest.mark.parametrize(
+    "list_name, list_type, suffix",
+    [
+        ("test css", "css", ".css"),
+        ("test html", "html", ".html"),
+        ("test txt", "text_plain", ".txt"),
+        ("test markdown", "markdown", ".md"),
+        ("test json", "json", ".json"),
+        ("test csv", "csv", ".txt"),
+    ],
+)
+def test_split_json_list(repo, list_name: str, list_type: str, suffix: str):
+    """
+    Given:
+        - A list json file.
+    When:
+        - Running `JsonSplitter.split_json` on it.
+    Then:
+        - Ensure the specific list dir created.
+        - Ensure the data file of the list with the extension matched has been created.
+        - Ensure the data section in the json list file is filled with `-`.
+    """
+    pack = repo.create_pack("PackName")
+    list_json = pack.create_list("test-list", LIST)
+    list_json.update({"id": list_name, "name": list_name, "type": list_type})
+    json_splitter = JsonSplitter(
+        input=list_json.path, output=pack.path, file_type=FileType.LISTS
+    )
+
+    folder_and_file_name = pascal_case(list_name)
+    expected_list_path = (
+        Path(pack.path) / "Lists" / folder_and_file_name / folder_and_file_name
+    ).with_suffix(".json")
+
+    with ChangeCWD(pack.repo_path):
+        res = json_splitter.split_json()
+        assert res == 0
+        assert expected_list_path.is_file()
+        assert (
+            (expected_list_path.parent / f"{folder_and_file_name}_data")
+            .with_suffix(suffix)
+            .is_file()
+        )
+        assert get_dict_from_file(str(expected_list_path))[0]["data"] == "-"
```

Comment:
use the new file service, let me know how it goes :)

Reply:
https://github.com/demisto/demisto-sdk/tree/master/demisto_sdk/commands/common/files